030-Concepts/080-pricing.mdx

```diff
@@ -7,26 +7,228 @@ slug: concepts/pricing
+
+- **Elasticsearch integration:** Integrates efficient and responsive full-text search capabilities within your projects. Our search engine uses Elasticsearch, which makes searches easy because it replicates all the data from our database. You get 15 GB of space initially, but can add more depending on what you need.
+
+- **File attachments:** Xata uses S3 and Cloudflare's CDN for storing and delivering files. Your files are stored safely and you can access them quickly when you need them. Starting storage is 2 GB, but you can add more or customize it per your needs.
+
+- **Image transformations:** You can do 10k image transformations from the start and add more as needed.
+
+- **AI questions:** Xata offers an option for integrating with OpenAI and ChatGPT, allowing you to ask up to 250 AI-powered questions. If you want to ask more questions, you can add more with the Pro or Enterprise plans.
+
+- **Support:** Depending on your plan, you get different levels of support, starting from community support to dedicated support.
+
+### Free plan
+
+The Free plan is perfect for individuals who are working on personal projects or are in the experimentation phase. This plan offers the basic functionalities needed to get started, at no cost.
+
+**The Free plan is ideal for:**
+
+- Individual developers
+- Students
+- Hobbyists
+- Experimenters who want to learn more about Xata
+
+##### Features
+
+- **Database branches:** Manage your relational and structural data using up to 10 database branches.
+- **Availability:** High availability to ensure your projects are always up and running, with minimal disruptions.
+- **Data storage:** 15 GB for storing the data related to your projects. This does not include search engine and file attachment storage.
+- **Search engine storage:** 15 GB for searching and storing results. You can use this space for enabling full-text search. Search engine storage currently bundles storage on disk with I/O rates and compute costs.
+- **File attachment storage:** 2 GB for storing file attachments.
+- **AI queries/month:** Run queries with AI integration up to 250 times a month.
+- **Support:** Community support is offered, so you can engage with community forums for assistance and discussions.
+
+### Pro plan
+
+The Pro Plan is optimized for small to medium-sized teams that require more resources. It operates on a usage-based pricing model, allowing you to scale resources as needed.
+
+Each billing period has a minimum charge of $20. If there are any additional charges related to the plan during the month, you will see those charges invoiced at the start of the following billing period.
+
+**The Pro plan is ideal for:**
+
+- Small to medium-sized teams
+- Startups
+- Growing projects
+
+> If you're affiliated with a startup or a non-profit organization, contact us. We have specific programs and discounts designed especially for startups and non-profits.
```

Comment:
I wonder if this warrants its own section. We get enough inquiries that it might be worth having more than a callout.